I.pass-tests/11-cities-formatter/cities.js

```diff
@@ -0,0 +1,4 @@
+function cities(array, func) {
+    return array.map(item => func(item))
```

Comment:
correct
also when writing map, you don't need to create it as array function if you're passign a function 
something like 
```array.map(func)```